src/main/java/racingcar/domain/car/Cars.java

```diff
@@ -24,17 +23,13 @@ private boolean isNullOrEmpty(List<Car> cars) {
         return cars == null || cars.isEmpty();
     }
 
-    public Cars(int carCount, CarActionStrategy carActionStrategy) {
-        validateCarCountsAndStrategy(carCount, carActionStrategy);
-        this.cars = new ArrayList<>();
-
-        for (int i = 0; i < carCount; i++) {
-            cars.add(new Car(carActionStrategy));
-        }
+    public Cars(List<String> carNames, CarActionStrategy carActionStrategy) {
```

Comment:
일급컬렉션 `CarNames` 를 구현하시는 건 어떨까요? 😄 
```suggestion
    public Cars(CarNames carNames, CarActionStrategy carActionStrategy) {
```